### DISCUSSION TITLE
phoenix.log file size

### DISCUSSION THREAD

**Discussion by FiliTol** [ACINQ/phoenixd#66] [Q&A] [answered]
Hi all!
My question relates to the phoenix.log file: the size is costantly increasing even though I'm not using my phoenixd much lately.

Is it safe to delete the phoenix.log file?

Thanks

**Comment by pm47** [comment, marked as answer]
Hello, yes you can delete the log file. Logging is quite basic at the moment, except if you are using the `jvm` distribution, which uses logback and can be fully configured with rotation, etc.

**Comment by bitcointwente** [comment]
@pm47 I had a VPS running with phoenixd for over year and it ran out of storage because my phoenix.log file was almost 10gb. I also have another instance running for over 1 month now, that one has a size of 3.6gb. Both instances run in combination with LNbits. Can't imagine that this is expected?

**Reply by pm47** [reply]
Hi, sorry missed this post. This issue was fixed in 0.5.0, where log files are now rolling. Please upgrade
